data/base-help.yaml

```diff
@@ -933,3 +933,9 @@ card_bags:
     - card-collector
   specific_descriptions:
     card-collector: Sets bag to draw cards from (fresh) and bag to place duplicates and your case(duplicates)
+    
+magic_training_room:
```

Comment:
Could you throw this setting alphabetically in this file? I missed that card-collector was put at the end, too. We can PR separately for that. This helps searching within the file itself, but it also allows a `;script-name help` to list the settings used alphabetically as well. Just keeps it kind of ordered. TY.

Reply:
Yes, apologies, i didnt realize they were alphabetical. Ill get it when i get back. Sorry about that

Reply:
> Yes, apologies, i didnt realize they were alphabetical. Ill get it when i get back. Sorry about that

No apology needed. It's a new thing, so it's ok.

Reply:
Is this comment resolved?